src/Facebook/Helpers/FacebookRedirectLoginHelper.php

```diff
@@ -211,13 +191,24 @@ public function getLoginUrl($redirectUrl,
    * @param string $separator The separator to use in http_build_query().
    *
    * @return string
+   *
+   * @throws FacebookSDKException
    */
   public function getLogoutUrl($accessToken, $next, $separator = '&')
```

Comment:
Yeah, I think this functionality is broken on FB's end, but I'd like to submit a bug request and double check it's no longer supported before killing it in the SDK. :)